.travis.yml

```diff
@@ -1,8 +1,8 @@
+dist: xenial
+services:
+  - xvfb
```

Comment:
https://docs.travis-ci.com/user/gui-and-headless-browsers/#using-xvfb-to-run-tests-that-require-a-gui